test/basic.test.tsx

```diff
@@ -1,82 +1,75 @@
-import * as React from 'react';
-import Downshift, { ControllerStateAndHelpers, StateChangeOptions } from '../';
+import * as React from 'react'
+import Downshift, { StateChangeOptions, DownshiftInterface } from '../'
+
+type Item = string
+const TypedDownShift: DownshiftInterface<Item> = Downshift
```

Comment:
Is there a better way to do this

Reply:
I tried to check how people solve this usually, and this seems to be the best possible solution as of now.